src/popup/App.vue

```diff
@@ -245,10 +250,23 @@ body {
   .tab-bar {
     position: sticky;
     bottom: 0;
+
+    @include mobile {
+      position: fixed;
```

Comment:
Looks too complicated, why don't use `position: fixed` in both mobile and extension? Then we will be able to remove `min-height: 600px` and similar